### PR TITLE
changed index.md and added picture file

### PR DESCRIPTION
**Pull request by adam-lam**

### CODE REVIEW

File: users/lamad/index.md [hunk 1]
```diff
@@ -6,12 +6,33 @@ author: Adam Lam
 Adam Lam
 ================================
 
-lamad@oregonstate.edu
+<img src="{{ site.url }}users/harterj/images/jrh-sit.jpg" height="375" width="600">
```
Comment (aaronjamesreynolds) [inline users/lamad/index.md]: Looks like you may be adding a picture of Jackson?

File: users/lamad/index.md [hunk 2]
```diff
@@ -6,12 +6,33 @@ author: Adam Lam
...
 
 
-## Research
-  Currently I am looking into Levermore-Pomraning Binary Stochastic Mixing for Transport in 3D as a comparison with Benchmark results.
+### Background
+I am currently a third year graduate student in Nuclear Engineering at OSU, working with Dr. Todd Palmer as an advisor and project leader. I got my undergraduate degree in Physics and Minor in Mathematics at California State University, Sacramento (CSUS). During my time at CSUS, I was a member of Zigma Pi Zigma society in Physics. I joined the team in the Fall of 2016 and have been working in the field of transport and reactor physics since.
+
+I have worked on two different project (stochastic mxing of materials, and multiband cross section generations) during my first two years with Dr. Palmer. I've also had the opportunity to interned at Lawrence Livermore National Laboratory for two summers (2017 and 2018).
```
Comment (aaronjamesreynolds) [inline users/lamad/index.md]: 'to interned' or 'to intern'?

File: users/lamad/index.md [hunk 3]
```diff
@@ -6,12 +6,33 @@ author: Adam Lam
...
+
+### Current projects
+* Master's Thesis
+  * I am in the process of finishing my thesis and looking to defend in May/June 2019
```
Comment (albeanth) [inline users/lamad/index.md]: a little more detail maybe? same with dissertation section below.

File: users/lamad/index.md [hunk 4]
```diff
@@ -6,12 +6,33 @@ author: Adam Lam
...
+
+* Dissertation
+  * Currently doing literature review and collecting material to write my dissertation proposal
+
+
+### Publications
+
+1. Coline Larmier, Adam Lam, Patrick Brantley, Fausto Malvagi, Todd Palmer, Andrea Zoia, ''Monte Carlo chord length sampling for d-dimensional Markov binary mixtures,'' J Quant. Spectrosc. Radiat. Transfer, 204 (2018), pp. Pages 256-271
```
Comment (albeanth) [inline users/lamad/index.md:30]: Can you leave links to these publications? (if you want...)

File: users/lamad/index.md [hunk 5]
```diff
@@ -6,12 +6,33 @@ author: Adam Lam
...
+
+2. Adam Lam, Todd Palmer, Jonathan Walsh, ''Demonstration of Multigroup Multiband Cross Section Generation in Monte Carlo Simulations,'' Journal of Computational Transport Theory. Pending
+
+### Research Interests
+1. Transport
```
Comment (albeanth) [inline users/lamad/index.md]: Radiation transport? Or automobile transport?

File: users/lamad/index.md (hunk 1 above)
Comment (albeanth) [reply users/lamad/index.md]: @aaronjamesreynolds do you not want a picture of Jackson on your page? 😁

File: users/lamad/index.md (hunk 3 above)
Comment (adam-lam) [reply users/lamad/index.md]: Will add more to the dissertation section once I get more details/information.

File: users/lamad/index.md (hunk 4 above)
Comment (adam-lam) [reply users/lamad/index.md:30]: Yea, I'll look into the links. Thanks.

File: users/lamad/index.md (hunk 5 above)
Comment (adam-lam) [reply users/lamad/index.md]: Maybe a bit of both? But I just changed it. I'll push a new one on now.

**Comment by aaronjamesreynolds** [comment]
Looks good to me. :shipit:

**Pull request merged**